src/zjs_performance.c

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2016, Linaro Limited.
+#ifdef BUILD_MODULE_PERFORMANCE
+
+// ZJS includes
+#include "zjs_util.h"
+
+static jerry_value_t zjs_performance_now(const jerry_value_t function_obj,
+                                         const jerry_value_t this,
+                                         const jerry_value_t argv[],
+                                         const jerry_length_t argc)
+{
+    if (argc != 0)
```

Comment:
I don't think you should check argc, because javascript allows you to pass in additional args, they'll just be ignored.